atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/jmx/CassandraJmxCompactionManager.java

```diff
@@ -100,9 +100,9 @@ private boolean executeInParallel(ExecutorService exec, List<? extends Callable<
 
         for (Future<Void> f : futures) {
             if (f.isCancelled()) {
-                log.error("Task execution timeouts in {} seconds. Timeout seconds:{}.", stopWatch.stop(), timeoutInSeconds);
-                throw new TimeoutException(String.format("Task execution timeouts in {} seconds. Timeout seconds:{}.",
-                        stopWatch.stop(), timeoutInSeconds));
+                log.error("Task execution timeouts in {} seconds. Timeout seconds:{}.", stopWatch.stop().elapsed(TimeUnit.SECONDS), timeoutInSeconds);
+                throw new TimeoutException(String.format("Task execution timeouts in %d seconds. Timeout seconds:%d.",
+                        stopWatch.elapsed(TimeUnit.SECONDS), timeoutInSeconds));
```

Comment:
Backported to where?  Are there old versions that are currently considered supported with backports?